Revamp query expression related BBEs (#3901)
ballerina-platform/ballerina-distribution: examples/nested-query-expressions/nested_query_expressions.md

# Nested query expressions

> Intermediate clauses of a query expression can contain another query expression and there is no limit for the amount of such nested query expressions. This is similar to nested foreach statements. Nested query expressions are useful to create complex query expressions. Execution will be happened according to the respective clauses.

Reviewer: ```suggestion
Intermediate clauses of a query expression can contain another query expression and there is no limit to the amount of such nested query expressions. This is similar to nested `foreach` statements. Nested query expressions are useful to create complex query expressions. Execution will happen according to the respective clauses.
```

::: code creating_tables_with_query.bal :::

::: out creating_tables_with_query.out :::

## Related links

> - [Query expressions - Ballerina by example](https://ballerina.io/learn/by-example/query-expressions)

Reviewer: ```suggestion
- [Query expressions](https://ballerina.io/learn/by-example/query-expressions)
```
Fix all occurrences.
